src/background/messaging/internal-methods/message-handler.ts

```diff
@@ -36,6 +36,11 @@ export async function internalBackgroundMessageHandler(
     case InternalMethods.AddressMonitorUpdated:
       await syncAddressMonitor(message.payload.addresses);
       break;
+    case InternalMethods.AccountChanged:
+      await chrome.storage.local.set({
+        ['stacksAddress']: message.payload.stacksAddress,
```

Comment:
also, name isn't super clear. It's a stacks address, but which? the current one?